src/ToolBox/SOS/NETCore/SOS.NETCore.csproj

```diff
@@ -12,6 +12,8 @@
     <NoStdLib>true</NoStdLib>
     <NoCompilerStandardLib>true</NoCompilerStandardLib>
     <UseOpenKey Condition="'$(UseOpenKey)'==''">true</UseOpenKey>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>true</DelaySign>
```

Comment:
For anything that is being signed with the Open key you cannot delay sign it because it is a full key and the signing lab will reject it.